docs/data/material/migration/migration-v4/migration-v4.md

```diff
@@ -71,19 +71,36 @@ If you need to support IE 11, check out our [legacy bundle](/material-ui/guides/
 
 ## Update React & TypeScript version
 
+### Update React
+
 The minimum supported version of React has been increased from v16.8.0 to v17.0.0.
 
+If you are using a React version below 17.0.0, update your packages to at least v14.11.2 for Material UI and v17.0.0 for React.
+
+With npm:
+
+```sh
+npm update @material-ui/core@^4.11.2 react@^17.0.0
```

Comment:
This comment doesn't run

<img width="858" alt="Screenshot 2023-02-13 at 20 43 15" src="https://user-images.githubusercontent.com/3165635/218558524-e2ea97b0-844a-4689-8c5e-feeefb3c9b62.png">

I think that it should be 

```suggestion
npm install @material-ui/core@^4.11.2 react@^17.0.0
```